scripts/scil_compute_local_tracking.py

```diff
@@ -13,6 +13,27 @@
 Algo 'prob': a direction drawn from the empirical distribution function defined
 from the SF.
 
+The local tracking algorithm can also run on the GPU using the --use_gpu
+option (experimental). Below is a list of known divergences between the
+CPU and GPU implementations:
+    * Backend: The CPU implementation uses DIPY's LocalTracking and the
+        GPU implementation uses an in-house OpenCL implementation.
+    * Algo: For the GPU implementation, the only available algorithm is
+        Algo 'prob'.
+    * GPU options: Nearest neighbour interpolation is implemented for
+        GPU tracking and can be enabled with '--sh_interp nearest'.
```

Comment:
Just to make sure. We should have the same default as local_tracking_cpu.py as much as possible.

Reply:
Default is trilinear but can be set to nearest only for gpu tracking (no nearest neighbour for local tracking)